diff -rup Adafruit_SSD1306-2.5.1/Adafruit_SSD1306.cpp Adafruit_SSD1306_mod/Adafruit_SSD1306.cpp
--- Adafruit_SSD1306-2.5.1/Adafruit_SSD1306.cpp	2022-01-12 23:00:39.000000000 +0100
+++ Adafruit_SSD1306_mod/Adafruit_SSD1306.cpp	2022-01-28 10:29:58.341601000 +0100
@@ -51,7 +51,9 @@
 #endif
 
 #include "Adafruit_SSD1306.h"
-#include "splash.h"
+#ifndef SSD1306_NO_SPLASH
+    #include "splash.h"
+#endif
 #include <Adafruit_GFX.h>
 
 // SOME DEFINES AND STATIC VARIABLES USED INTERNALLY -----------------------
diff -rup Adafruit_SSD1306-2.5.1/Adafruit_SSD1306.h Adafruit_SSD1306_mod/Adafruit_SSD1306.h
--- Adafruit_SSD1306-2.5.1/Adafruit_SSD1306.h	2022-01-12 23:00:39.000000000 +0100
+++ Adafruit_SSD1306_mod/Adafruit_SSD1306.h	2022-01-28 10:29:58.357600000 +0100
@@ -33,7 +33,7 @@
 // AND HEIGHT ARGUMENTS).
 
 // Uncomment to disable Adafruit splash logo
-//#define SSD1306_NO_SPLASH
+#define SSD1306_NO_SPLASH
 
 #if defined(ARDUINO_STM32_FEATHER)
 typedef class HardwareSPI SPIClass;
Seulement dans Adafruit_SSD1306-2.5.1: splash.h
