diff -rup ComponentTester-1.50m-000/config_328.h ComponentTester-1.50m-git/config_328.h
--- ComponentTester-1.50m-000/config_328.h	2023-09-28 15:30:35.098277201 +0200
+++ ComponentTester-1.50m-git/config_328.h	2023-09-27 06:51:50.895755995 +0200
@@ -44,7 +44,7 @@
 #define LCD_HD44780                     /* display controller HD44780 */
 #define LCD_TEXT                        /* character display */
 #define LCD_PCF8574                     /* PCF8574 backpack */
-#define LCD_I2C_ADDR     0x3f           /* PCF8574's I2C address */
+#define LCD_I2C_ADDR     0x27           /* PCF8574's I2C address */
 /* control and data lines */
 #define LCD_DB_STD                      /* use standard pins 4-7 for DB4-7 */
 #define LCD_DB4          PCF8574_P4     /* port pin used for DB4 */
@@ -65,11 +65,12 @@
 /* I2C bus */
 #define I2C_BITBANG                     /* bit-bang I2C */
 #define I2C_STANDARD_MODE               /* 100kHz bus speed */
-#define I2C_PORT         PORTD          /* I2C port data register */
-#define I2C_DDR          DDRD           /* I2C port data direction register */
-#define I2C_PIN          PIND           /* I2C port input pins register */
-#define I2C_SDA          PD0            /* port pin used for SDA */
-#define I2C_SCL          PD1            /* port pin used for SCL */
+#define I2C_PORT         PORTC          /* I2C port data register */
+#define I2C_DDR          DDRC           /* I2C port data direction register */
+#define I2C_PIN          PINC           /* I2C port input pins register */
+#define I2C_SDA          PC4            /* port pin used for SDA */
+#define I2C_SCL          PC5            /* port pin used for SCL */
+
 
 
 /* ************************************************************************
@@ -92,8 +93,8 @@
 #define TP3              PC2       /* test pin / probe #3 */
 
 #define TP_ZENER         PC3       /* test pin for for Zener check (10:1 voltage divider) */
-#define TP_REF           PC4       /* test pin for 2.5V reference and relay */
-#define TP_BAT           PC5       /* test pin for battery (4:1 voltage divider) */
+#define TP_REF           PC6       /* test pin for 2.5V reference and relay */
+#define TP_BAT           PC7       /* test pin for battery (4:1 voltage divider) */
 #define TP_LOGIC         PC3       /* test pin for Logic Probe (4:1 voltage divider) */
 
 
diff -rup ComponentTester-1.50m-000/config.h ComponentTester-1.50m-git/config.h
--- ComponentTester-1.50m-000/config.h	2023-09-27 03:23:05.788589715 +0200
+++ ComponentTester-1.50m-git/config.h	2023-09-27 23:07:38.044307088 +0200
@@ -847,7 +847,7 @@
  *  - uncomment to enable
  */
 
-//#define UI_AUTOHOLD
+#define UI_AUTOHOLD
 
 
 /*
@@ -924,7 +924,7 @@
  *    until it's powered off manually.
  */
 
-#define CYCLE_MAX        5
+#define CYCLE_MAX        255
 
 
 /*
@@ -1177,8 +1177,8 @@
  *  - enable one
  */
 
-#define POWER_SWITCH_SOFT
-//#define POWER_SWITCH_MANUAL
+//#define POWER_SWITCH_SOFT
+#define POWER_SWITCH_MANUAL
 
 
 /*
@@ -1189,7 +1189,7 @@
  *  - uncomment one of the modes
  */
 
-//#define BAT_NONE
+#define BAT_NONE
 //#define BAT_DIRECT
 #define BAT_DIVIDER
 
@@ -1252,7 +1252,7 @@
  *  - uncomment to enable
  */
 
-#define SAVE_POWER
+//#define SAVE_POWER
 
 
 
@@ -1286,7 +1286,7 @@
  */
 
 /* Rl in Ohms */
-#define R_LOW            680
+#define R_LOW            670
 
 /* Rh in Ohms */
 #define R_HIGH           470000
@@ -1385,7 +1385,7 @@
  *  - uncomment to enable
  */
 
-//#define ADC_LARGE_BUFFER_CAP
+#define ADC_LARGE_BUFFER_CAP
 
 
 
diff -rup ComponentTester-1.50m-000/main.c ComponentTester-1.50m-git/main.c
--- ComponentTester-1.50m-000/main.c	2023-09-27 03:23:05.944589715 +0200
+++ ComponentTester-1.50m-git/main.c	2023-09-27 23:18:39.503636790 +0200
@@ -3081,10 +3081,12 @@ cycle_control:
   {
     Key = KEY_MAINMENU;            /* signal main menu */
   }
+  #if 0
   else if (Key == KEY_LONG)        /* long key press */
   {
     Key = KEY_POWER_OFF;           /* signal power off */
   }
+  #endif
   #ifdef HW_KEYS
   else if (Key == KEY_LEFT)        /* rotary encoder: left turn */
   {
diff -rup ComponentTester-1.50m-000/Makefile ComponentTester-1.50m-git/Makefile
--- ComponentTester-1.50m-000/Makefile	2023-09-27 03:23:05.756589715 +0200
+++ ComponentTester-1.50m-git/Makefile	2023-09-27 01:11:00.846399528 +0200
@@ -29,7 +29,7 @@ MCU = atmega328
 # - 8MHz  : 8
 # - 16MHz : 16
 # - 20MHz : 20
-FREQ = 8
+FREQ = 16
 
 # oscillator type
 # - internal RC oscillator      : RC
@@ -84,9 +84,9 @@ PARTNO = m328p
 #OPTIONS = -B 10.0
 
 # Diamex ALL-AVR/AVR-Prog
-PROGRAMMER = avrispmkII
-PORT = usb
-OPTIONS = -B 1.0
+#PROGRAMMER = avrispmkII
+#PORT = usb
+#OPTIONS = -B 1.0
 
 # Pololu USB AVR Programmer
 #PROGRAMMER = stk500v2
@@ -103,10 +103,15 @@ OPTIONS = -B 1.0
 #PORT = usb
 #OPTIONS = -B 5.0
 
-# Arduino Uno bootloader via serial/USB
-#PROGRAMMER = arduino
-#PORT = /dev/ttyACM0
-#OPTIONS = -D -b 115200
+# Arduino Uno
+#~ PROGRAMMER = arduino
+#~ PORT = /dev/ttyACM0
+#~ OPTIONS = -D -b 115200
+
+# Arduino Pro Mini
+PROGRAMMER = arduino
+PORT = /dev/ttyUSB0
+OPTIONS = -D -b 57600
 
 # Arduino Mega2560 bootloader via serial/USB
 #PROGRAMMER = wiring
